Storage/SQL/BrowserTable.swift

```diff
@@ -262,14 +266,34 @@ open class BrowserTable: Table {
             "provider_name TEXT, " +
             "created_at DATETIME DEFAULT CURRENT_TIMESTAMP, " +
             "expired_at LONG" +
-        ") "
+    ") "
+
+    let attachedPageMetadataCreate =
+        "CREATE TABLE IF NOT EXISTS \(AttachedTablePageMetadata) (" +
+            "id INTEGER PRIMARY KEY, " +
+            "cache_key LONGVARCHAR UNIQUE, " +
+            "site_url TEXT, " +
+            "media_url LONGVARCHAR, " +
+            "title TEXT, " +
+            "type VARCHAR(32), " +
+            "description TEXT, " +
+            "provider_name TEXT, " +
+            "created_at DATETIME DEFAULT CURRENT_TIMESTAMP, " +
+            "expired_at LONG" +
+    ") "
 
     let indexPageMetadataCacheKeyCreate =
     "CREATE UNIQUE INDEX IF NOT EXISTS \(IndexPageMetadataCacheKey) ON page_metadata (cache_key)"
 
     let indexPageMetadataSiteURLCreate =
     "CREATE UNIQUE INDEX IF NOT EXISTS \(IndexPageMetadataSiteURL) ON page_metadata (site_url)"
 
+    let attachedIndexPageMetadataCacheKeyCreate =
+    "CREATE UNIQUE INDEX IF NOT EXISTS \(AttachedIndexPageMetadataCacheKey) ON \(TablePageMetadata) (cache_key)"
+
+    let attachedIndexPageMetadataSiteURLCreate =
+    "CREATE UNIQUE INDEX IF NOT EXISTS \(AttachedIndexPageMetadataSiteURL) ON \(TablePageMetadata) (site_url)"
```

Comment:
No, interestingly that is [not how you do indexes on an attached table](http://sqlite.1065341.n5.nabble.com/creating-indexes-in-attached-databases-td13100.html). You add the attached DB as a reference on the index name, and refer to the table directly. Odd huh?